packages/extension-api/src/extension-api.d.ts

```diff
@@ -539,14 +546,17 @@ declare module '@podman-desktop/api' {
 
   // create programmatically a ContainerProviderConnection
   export interface ContainerProviderConnectionFactory extends ProviderConnectionFactory {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    create(params: { [key: string]: any }, logger?: Logger, token?: CancellationToken): Promise<void>;
+    create(params: { [key: string]: unknown }, logger?: Logger, token?: CancellationToken): Promise<void>;
```

Comment:
I'm sorry you can't change here as it would be a non backward compatible change, only for the new interfaces 

Ping @benoitf for confirmation